h5c/vic/src/vic-webapp/src/app/create-vch-wizard/compute-capacity/compute-capacity.component.ts

```diff
@@ -178,60 +196,87 @@ export class ComputeCapacityComponent implements OnInit {
         });
     }
 
-    // update resource limit & reservation info
-    this.createWzService.getResourceAllocationsInfo(resourceObjForResourceAllocations, isCluster)
-    .subscribe(response => {
-      const cpu = response['cpu'];
-      const memory = response['memory'];
-      this.resourceLimits = response;
-
-      // set max limit validator for cpu maxUsage
-      this.form.get('cpuLimit').setValidators([
-        ...getNumericValidatorsArray(true),
-        Validators.max(cpu['maxUsage'])
-      ]);
-
-      // set max limit validator for memory maxUsage
-      this.form.get('memoryLimit').setValidators([
-        ...getNumericValidatorsArray(true),
-        Validators.max(memory['maxUsage'])
-      ]);
+    const vmGroupsObs: Observable<any[]> = isCluster ?
```

Comment:
add small comment here. like... if its a cluster, get the cluster vm groups, otherwise set to empty array observable...

just faster when to read it that way when scanning the code